cookiecutter.json

```diff
@@ -1,5 +1,10 @@
 {
-    "directory_name": "project-name",
+    "_extensions": ["local_extensions.GithubExtension"],
+    "project_name": "Project Name",
+    "directory_name": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
     "author_name": "Your Name",
+    "dependency_manager": ["pip", "poetry"],
+    "data_version_control": ["yes", "no"],
+    "__dep_filename": "{{ cookiecutter.dependency_manager|dep_filename }}",
     "compatible_python_versions": "^3.8"
 }
```

Comment:
> I would avoid asking for directory_name as it can be achieved with project_name

Agreed.

Reply:
> I would avoid asking for directory_name as it can be achieved with project_name.

Agreed.

Reply:
> What if we just provide data version control by default for now?

Can putting DVC as a default cause any conflict with another tool? I have no experience with DVC yet, so I don't know.

Reply:
> Selecting data_version_control with the options "yes" or "no" is a bit weird.

Yeah, it is weird :(

Reply:
> > What if we just provide data version control by default for now?
> 
> My main idea is avoid imposing anything to the user. Maybe someone might not want to use DVC (that is my case at the moment). However, if it is temporary, for me, okay.
> Agreed
> Can putting DVC as a default cause any conflict with another tool? I have no experience with DVC yet, so I don't know.
> Nope. It will have no conflict with other tools.